pulsar-client/src/main/java/org/apache/pulsar/client/impl/conf/ClientConfigurationData.java

```diff
@@ -124,6 +124,19 @@ public class ClientConfigurationData implements Serializable, Cloneable {
     )
     private int connectionsPerBroker = 1;
 
+    @ApiModelProperty(
+            name = "connectionMaxIdleSeconds",
+            value = "Release the connection if it is not used for more than [connectionMaxIdleSeconds] seconds. "
+                    + "If  [connectionMaxIdleSeconds] < 0, disabled the feature that auto release the idle connections"
+    )
+    private int connectionMaxIdleSeconds = 180;
+
+    @ApiModelProperty(
+            name = "connectionIdleDetectionIntervalSeconds",
+            value = "How often check idle connections"
+    )
+    private int connectionIdleDetectionIntervalSeconds = 60;
```

Comment:
The `connectionMaxIdleSeconds` should always >= `connectionIdleDetectionIntervalSeconds`? and is it able to remove this configuration? Instead, we can have an interval second based on the  `connectionMaxIdleSeconds` to simplify the configuration.

Reply:
> The connectionMaxIdleSeconds should always >= connectionIdleDetectionIntervalSeconds?

no such limit.

>  is it able to remove this configuration? Instead, we can have an interval second based on the connectionMaxIdleSeconds to simplify the configuration.

Good idea. Already fixed.